### DISCUSSION TITLE
Include Container Insight Metrics in ADOT Config

### DISCUSSION THREAD

**Discussion by joshuscurtis** [artilleryio/artillery#2656] [Artillery]
Would it be possible to enable the AWS ECS Container Metrics Receiver so that we can easily get metrics on how the containers are performing?

[ECS Metrics Receiver](https://aws-otel.github.io/docs/components/ecs-metrics-receiver)

**Comment by InesNi** [comment]
Hi @joshuscurtis,

That's a good shout, thanks! We'll add it to the list for feature requests, and look into it  🙌🏻
